packages/jest-runner/src/__tests__/testRunner.test.ts

```diff
@@ -6,9 +6,9 @@
  *
  */
 
-import {TestWatcher} from '@jest/core';
```

Comment:
do we still need the dep on `@jest/core`?

Reply:
Seems like `jest-runner` did not list it as a dependency 🤷‍♂️

Reply:
> do we still need the dep on `@jest/core`?

![Screenshot_20220408-034121](https://user-images.githubusercontent.com/97814333/162417506-0aaadace-0051-496c-b0c4-01bfcde8972f.png)

Reply:
oh, haha